parlai/agents/transformer/modules/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.agents.transformer.modules.generator import *  # noqa: F401
```

Comment:
Ha, this reminds me of my MATLAB days (yes I am old) where people used to ["poof"](https://www.reddit.com/r/matlab/comments/f2rx2m/what_is_poofing_variables/) variables into a single huge namespace, making it hard to tell where they came from. But here I suppose it's a good idea to maintain backwards compatibility, at least in the near-term

Reply:
Ya can we not import *?